osu.Game.Rulesets.Taiko/TaikoRuleset.cs

```diff
@@ -208,6 +208,51 @@ public override IEnumerable<Mod> GetModsFor(ModType type)
 
         public ILegacyScoreSimulator CreateLegacyScoreSimulator() => new TaikoLegacyScoreSimulator();
 
+        public double GetLegacyScoreMultiplier(IReadOnlyList<Mod> mods, LegacyBeatmapConversionDifficultyInfo difficulty)
+        {
+            bool scoreV2 = mods.Any(m => m is ModScoreV2);
+
+            double multiplier = 1.0;
+
+            foreach (var mod in mods)
+            {
+                switch (mod)
+                {
+                    case TaikoModNoFail:
+                        multiplier *= scoreV2 ? 1.0 : 0.5;
+                        break;
+
+                    case TaikoModEasy:
+                        multiplier *= 0.5;
+                        break;
+
+                    case TaikoModHalfTime:
+                    case TaikoModDaycore:
+                        multiplier *= 0.3;
+                        break;
+
+                    case TaikoModHidden:
+                        multiplier *= 1.06;
+                        break;
+
+                    case TaikoModHardRock:
+                    case TaikoModFlashlight:
+                        multiplier *= 1.12;
+                        break;
```

Comment:
That... doesn't look right?

https://github.com/peppy/osu-stable-reference/blob/1531237b63392e82c003c712faa028406073aa8f/osu!/GameplayElements/Scoring/ModManager.cs#L249-L250
https://github.com/peppy/osu-stable-reference/blob/1531237b63392e82c003c712faa028406073aa8f/osu!/GameplayElements/Scoring/ModManager.cs#L253-L254

Shouldn't HR be 1.06x?